tests/Unit/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Test_TovStar.cpp

```diff
@@ -79,15 +53,35 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticSolutions.RelEuler.Tov",
           "TovStar:\n"
           "  CentralDensity: 1.0e-3\n"
           "  PolytropicConstant: 100.0\n"
-          "  PolytropicExponent: 2.0\n");
+          "  PolytropicExponent: 2.0\n"
+          "  Coordinates: " +
+          get_output(coord_system));
   const auto deserialized_option_solution =
       serialize_and_deserialize(option_solution);
   const auto& solution =
       dynamic_cast<const RelativisticEuler::Solutions::TovStar&>(
           *deserialized_option_solution);
+  {
+    INFO("Semantics");
+    CHECK(solution == TovStar{1.0e-3, 100.0, 2.0, coord_system});
+    test_copy_semantics(solution);
+    TovStar star_copy = solution;
+    test_move_semantics(std::move(star_copy), solution);
+    test_serialization(solution);
+  }
+  if (coord_system == TovCoordinates::Schwarzschild) {
+    INFO("Properties");
+    Approx custom_approx = Approx::custom().epsilon(1.0e-08).scale(1.0);
+    CHECK(solution.radial_solution().outer_radius() ==
+          custom_approx(10.0473500683));
+    // Check a second solution
+    TovStar second_solution{1.0e-3, 8.0, 2.0};
+    CHECK(second_solution.radial_solution().outer_radius() ==
+          custom_approx(3.4685521362));
+  }
 
-  const double radius_of_star = 10.04735006833273303;
-  CHECK(solution.radial_solution().outer_radius() == approx(radius_of_star));
+  // Check the solution numerically solves the GRMHD equations
+  const double radius_of_star = solution.radial_solution().outer_radius();
```

Comment:
I test this in a follow-up PR with the XCTS equations, where I compute Einstein constraints in various ways.

Note that the GH system doesn't have matter terms coded up, so it's nontrivial to add a quick test of the GH system here. None of the other GR matter solutions do that either AFAICT, probably for the same reason. I think this would be useful to add, but seems out of scope for this PR. Again, I test Einstein constraints in the next PR.